npc/jobs/2-2/crusader.txt

```diff
@@ -223,6 +209,27 @@ prt_castle,45,169,5	script	Senior Crusader	4_M_CRU_OLD,{
 	else if(CRUS_Q >= 1 && CRUS_Q <= 3) {
 		mes "Have you proven your determination with the task I have given you, or do you possess the items proving that you have received your calling?";
 		next;
+		if (countitem(Patriotism_Marks) >= 1 && countitem(Sacred_Marks) >= 1) {
+			mes "Ah...";
+			mes "I see that you have been called to become a Crusader. We are assured of your will, but now we must test your capabilities.";
+			next;
+			mes "[Michael Halig]";
+			mes "Meet with Moorenak Miyol who is training in the underground dungeon of the Prontera Castle. Go, and speak with him first.";
+			next;
+			delitem Patriotism_Marks, 1;
+			delitem Sacred_Marks, 1;
+			CRUS_Q = 4;
+			if (questprogress(3006)) {
+				changequest 3006,3009;
+			} else if (questprogress(3007)) {
+				changequest 3007,3009;
+			} else {
+				changequest 3008,3009;
```

Comment:
I see what you mean now. I find it indeed weird, why all scripts use this "removal behavior" instead of preserving questlog, which is kind of the point of a log.

But I'd suggest that we won't change this for this PR, as it'd require me to change it everywhere in that script. I'd like to separate that in another PR if I were to do that, since I am not quite sure about the stance of Hercules staff on this.

(I was actually quite close of standardizing the code-style and syntax of the whole script.
But decided against it, as it'd slow down the PR process for the actual fix I want.)